src/core/brand/roundel-brand.tsx

```diff
@@ -1,6 +1,7 @@
 import React from "react"
 import { SvgRoundelFilled } from "./roundel-filled"
+import { brand } from "@guardian/src-foundations/palette"
```

Comment:
This is a great idea! Would it be possible for you to add `@guardian/src-foundations` as a `peerDependency` of this package?

Reply:
Thanks, I’ll have a look at that!